platform/android/os_android.h

```diff
@@ -86,6 +86,8 @@ class OS_Android : public OS_Unix {
 
 public:
 	static const char *ANDROID_EXEC_PATH;
+	static const int DEFAULT_WINDOW_WIDTH = 800;
```

Comment:
I'm not sure the editor can be used at all in 800x600. And for the project itself the default size seems to be bigger than that (at least for the project setting). Maybe we should bump this one too?

Reply:
Bumping should be fine, and the default dimensions can be made to match the editor min dimensions: `1024x600`.

Since the application is landscape, the dimension that matters the most is the height. Most Android devices have a 16:9 aspect ratio, so the min height can be updated to `500`. From my testing, the editor UX still fits comfortably within that frame. Lowering the min height also increases the largest supported scale which improves readability.